src/block_cache/block_mgr.c

```diff
@@ -8,7 +8,6 @@
 
 #include "wt_internal.h"
 
-static void __bm_method_set(WT_BM *, bool);
```

Comment:
Rather than changing this to have `__wt_bm_method_set`, I would recommend creating a `__ut_bm_method_set` function under a #IFDEF e.g.
```
#ifdef HAVE_UNITTEST
int
__ut_block_write_off(WT_SESSION_IMPL *session, WT_BLOCK *block, WT_ITEM *buf, wt_off_t *offsetp,
  uint32_t *sizep, uint32_t *checksump, bool data_checksum, bool checkpoint_io, bool caller_locked)
{
    return (__block_write_off(
      session, block, buf, offsetp, sizep, checksump, data_checksum, checkpoint_io, caller_locked));
}
#endif
```

Reply:
I've addressed this [here](https://github.com/wiredtiger/wiredtiger/pull/11010/commits/41c6b93e0922df0854d4a7dc1c360885a24da9b2).